dev/wlp-jakartaee-transform/rules/jakarta-renames.properties

```diff
@@ -234,3 +247,14 @@ org.eclipse.persistence.sessions=org.eclipse.persistence.sessions
 org.eclipse.persistence.tools.schemaframework=org.eclipse.persistence.tools.schemaframework
 org.eclipse.persistence.transaction=org.eclipse.persistence.transaction
 
+com.sun.xml.internal.messaging.saaj=com.sun.xml.messaging.saaj
```

Comment:
These may not be needed.

Reply:
I see you removed one of the saaj renames, but the left the others.  My comment was suggesting that none of them may be needed.  I had put it in trying to get things working and once I had things working more I am not sure if it was needed any longer.  I suggest removing all of the rename s for com.sun.xml.internal.messaging.saaj.*